### ISSUE TITLE
AVT 3 - React Loading Component, VO/iOS and JAWS/Windows do not announce Loading

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#3446]
## Environment
Windows 10 
FireFox Quantum 68.0
JAWS 18

iOS 12.3.1
Safari
VoiceOver

## Description
When page loads "Active Loading Indicator" is not announced by screen readers.
Expected Result: Screen reader should announce "Active Loading Indicator" on page load. 

Note: This is working with VoiceOver on macOS.
